libdirections-offboard/src/main/java/com/mapbox/navigation/route/offboard/MapboxOffboardRouter.kt

```diff
@@ -39,6 +40,7 @@ class MapboxOffboardRouter(
     ) {
         mapboxDirections = RouteBuilderProvider.getBuilder(accessToken, context, skuTokenProvider)
             .routeOptions(routeOptions)
+            .enableRefresh(routeOptions.profile() == DirectionsCriteria.PROFILE_DRIVING_TRAFFIC)
```

Comment:
I have a couple ideas of a better place to enable this.

1. The customer will want to use it to update their traffic route line. Enable near where they use it
2. Bundle this feature as part of the `PROFILE_DRIVING_TRAFFIC`
3. Enable it close to the class that controls it

Dream 4. Server configuration where a customer can configure it with their access token

Reply:
@andrewychen do we have a side dashboard where we customers can configure their navigation sdk for their access tokens?